src/webview_provider/altimateWebviewProvider.ts

```
@@ -360,7 +364,7 @@ export class AltimateWebviewProvider implements WebviewViewProvider {
           });
           break;
         case "validateCredentials":
-          const isValid = await this.altimateRequest.handlePreviewFeatures();
+          const isValid = this.altimateAuthService.handlePreviewFeatures();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)